### ISSUE TITLE
Crash when having profiler open with "Only Active" and switching scenes

### ISSUE THREAD

**Issue by Lordmau5** [exeldro/obs-source-profiler#4]
Managed to get my GPU time down already so thank you for this plugin!

Had this happen just now when I tried switching scenes while it was active and I had a scene / source selected in the profiler that isn't on the new scene I transitioned to
![image](https://github.com/user-attachments/assets/7f747622-45e1-41cf-b2f8-edbd14256d46)

[Crash 2024-10-23 11-31-58.txt](https://github.com/user-attachments/files/17488483/Crash.2024-10-23.11-31-58.txt)
[2024-10-23 11-26-56.txt](https://github.com/user-attachments/files/17488487/2024-10-23.11-26-56.txt)

**Comment by exeldro** [comment]
I tried to replicate it my development environment and on my production environment, but have not been able to replicate the issue yet.
The crash report is not helping me much this time.

**Comment by Lordmau5** [comment]
Yeah I had a glance at the logs as well and nothing popped out to me either about this :(

I'll try to reproduce it tomorrow - I switched from the Games scene to the Chatting scene

Additionally I have a stinger transition that properly transitions at around 1s into it so maybe it's something to do with that?
